experimental/traffic-portal/src/app/core/servers/servers-table/servers-table.component.ts

```diff
@@ -320,23 +347,27 @@ export class ServersTableComponent implements OnInit {
 	public fuzzySubject: BehaviorSubject<string>;
 
 	/** Form controller for the user search input. */
-	public fuzzControl: UntypedFormControl = new UntypedFormControl("");
+	public fuzzControl: FormControl = new FormControl("");
 
 	/**
 	 * Constructs the component with its required injections.
 	 *
+	 * @param auth The user authorization service.
 	 * @param api The Servers API which is used to provide row data.
 	 * @param route A reference to the route of this view which is used to set the fuzzy search box text from the 'search' query parameter.
-	 * @param router Angular router
 	 * @param navSvc Manages the header
+	 * @param cdn The CDN API
 	 * @param dialog Dialog manager
 	 */
```

Comment:
fwiw, you can just get rid of this. With the switch from TSLint to `typescript-eslint` it became possible to not require JSDoc comments on constructors.